tests/src/Kernel/Paragraphs/LinksBlockTest.php

```diff
@@ -0,0 +1,386 @@
+<?php
+
+declare(strict_types=1);
```

Comment:
```suggestion
declare(strict_types = 1);
```
use drupal formatting